src/sagemaker/image_uri_config/blazingtext.json

```diff
@@ -1,7 +1,7 @@
 {
     "scope": ["inference", "training"],
     "versions": {
-        "1": {
```

Comment:
This should remain as `1` and we should add `latest` as an alias to `1`.  That's the policy for all 1P algorithms.